airship_tempest_plugin/hacking/checks.py

```diff
@@ -176,7 +176,7 @@ def no_rbac_test_suffix_in_test_class_name(physical_line, filename):
     """Check that RBAC class names end with "RbacTest"
     P102
     """
-    if "airship_tempest_plugin/tests/api/rbac/" in filename:
+    if "airship_tempest_plugin/tests/api/shipyard/rbac" in filename:
```

Comment:
it shouldn't be just for shipyard as more rbac tests for other services could be added later

Reply:
ok, but maybe use regex so it is not hardcoded to only shipyard.

Reply:
this still needs to be updated. nothing here should be hardcoded to shipyard